requirements/docs.txt

```diff
@@ -151,7 +151,7 @@ sphinxext-opengraph==0.8.2
     # via -r requirements/docs.in
 tornado==6.3.1
     # via livereload
-urllib3==1.26.15
+urllib3==2.0.2
```

Comment:
Lot of things changed here. This may be a little more trickier to review: https://github.com/urllib3/urllib3/blob/main/CHANGES.rst#200-2023-04-26

Reply:
We are not using it directly and `requests` was also upgraded, so I think we should be fine.

Reply:
This is a good guide https://urllib3.readthedocs.io/en/latest/v2-migration-guide.html#what-are-the-important-changes